extensions/amp-analytics/0.1/events.js

```diff
@@ -203,6 +203,22 @@ export function getTrackerTypesForParentType(parentType) {
   return filtered;
 }
 
+/**
+ * Expand the event variables to include default data-vars
+ * @param {!Element} target
+ * @param {!JsonObject} eventVars
+ * @return {!JsonObject}
+ */
+function mergeDataVars(target, eventVars) {
+  const vars = getDataParamsFromAttributes(
+    target,
+    /* computeParamNameFunc */ undefined,
+    VARIABLE_DATA_ATTRIBUTE_KEY
+  );
+  deepMerge(vars, eventVars, 1);
```

Comment:
How expensive is this on average?

Reply:
Also, does the order of the arguments for `deepMerge` describe whose property gets precedence in the case of a collision? I know we had discussed we want components to be able to override data vars, can you leave a comment to that effect either here or in the function comment?

Reply:
Depends. It's not expensive here for two reasons 
1. The depth is set to 1 (Actually 0 is sufficient, I switched to 0). No recursion
2. The vars and eventVars don't contain nested objects, no recursion. 

`#deepMerge(target, source, depth)` merges source to target. value from `eventVars` will override `vars`. Added comment.